support/runner-script.py

```
@@ -187,7 +205,11 @@ def run_upload(args):
 	import subprocess
 
 	proc = subprocess.run(
-		['gcloud', 'auth', 'print-access-token'], check=True, text=True, capture_output=True
+		['gcloud', 'auth', 'print-access-token'],
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)